src/fuzz/fuzzer-entry-point.c

```diff
@@ -0,0 +1,5 @@
+#include "fuzz.h"
+
+int FuzzerEntrypoint(const uint8_t *data, size_t size) {
+        return LLVMFuzzerTestOneInput(data, size);
+}
```

Comment:
Maybe we could convince oss-fuzz to switch to `FuzzerEntrypoint`? The more general name seems better.

Reply:
It'd make ClusterFuzz and FuzzBuzz a little bit more compatible with each other but `libFuzzer`, which a lot of people use locally, expects to get `LLVMFuzzerTestOneInput` so it's `libFuzzer` that should be changed first and I'm not sure whether the developers are ready to do that (and even if they are it's likely to break something on ClusterFuzz anyway :-). @everestmz I'm wondering if you tried to discuss this with someone from the OSS-Fuzz and/or libFuzzer teams. By the way, it'd be great if you could take a look at this PR too.

Reply:
We decided to use `FuzzerEntrypoint` since we're not just fuzzing C/C++ and it helped to keep everything a bit more unified, but I'm not sure if asking ClusterFuzz/libFuzzer to change is the right thing to do, especially since we're the new ones on the block here. Maybe we can work out a way to support both method names for C/C++. We support generic fuzzer method names for Go & Python, so it would make sense to do the same for C/C++.